manifests/app.pp

```diff
@@ -103,8 +103,9 @@
       active  => true,
     }
     ~> service { "unicorn_${name}":
-      ensure => running,
-      enable => true,
+      ensure  => running,
+      enable  => true,
+      restart => "/bin/systemctl reload-or-restart unicorn_${name}.service"
```

Comment:
Looks like the new param just needs a trailing comma and it'll be good to go!
```suggestion
      restart => "/bin/systemctl reload-or-restart unicorn_${name}.service",
```

Reply:
Oh gotcha, thanks! I pushed that change, so I'm going to mark this as "ready for review" now.